SpatialGDK/Source/SpatialGDK/Private/Tests/RPCContainerTest.cpp

```diff
@@ -0,0 +1,232 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "TestDefinitions.h"
+
+#include "Utils/RPCContainer.h"
+#include "Schema/RPCPayload.h"
+#include "MockObject.h"
+
+#include "Core.h"
+
+#if WITH_DEV_AUTOMATION_TESTS
+
+using namespace SpatialGDK;
+
+namespace
+{
+	static ESchemaComponentType ReliableType = ESchemaComponentType::SCHEMA_ClientReliableRPC;
+	static ESchemaComponentType UnreliableType = ESchemaComponentType::SCHEMA_ClientUnreliableRPC;
+	static ESchemaComponentType ServerReliableType = ESchemaComponentType::SCHEMA_ServerReliableRPC;
+	static ESchemaComponentType ServerUnreliableType = ESchemaComponentType::SCHEMA_ServerUnreliableRPC;
+	static ESchemaComponentType MulticastType = ESchemaComponentType::SCHEMA_NetMulticastRPC;
+	static ESchemaComponentType CrossServerType = ESchemaComponentType::SCHEMA_CrossServerRPC;
+
+	static FUnrealObjectRef GenerateObjectRef(UObject* TargetObject)
```

Comment:
Complete aside: what do these statics in this anonymous namespace actually do?

Reply:
It is, I will remove static keyword.

Reply:
Removed static keyword.